javascript/lifecycle.js

```diff
@@ -125,17 +126,29 @@ document.addEventListener(
 // - element - the element that triggered the reflex (not necessarily the Stimulus controller's element)
 //
 export const dispatchLifecycleEvent = (stage, element, reflexId) => {
+  const { reflexData, reflexController } = element
+
+  if (!document.body.contains(element)) {
+    const attrs = extractElementAttributes(element)
+    element = findElement(attrs, true)
```

Comment:
[I see you.](https://github.com/hopsoft/stimulus_reflex/pull/386/files#r531824146) 👁️